specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3635 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Routing Service",
+    "description": "Azure Communication Routing Service",
+    "version": "2021-10-20-preview2"
+  },
+  "paths": {
+    "/routing/classificationPolicies": {
+      "post": {
```

Comment:
The [Azure REST API Guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md) recommend using `PUT` or `PATCH` for resource creation. Is there a particular reason you decided to go with `POST`?

Reply:
This is largely to not use resource ids provided by customers which are potentially PII data. There is a thread related to this b/w BGA and JEFFREYR.